### DISCUSSION TITLE
Open text in external text editor?

### DISCUSSION THREAD

**Discussion by azzamsa** [gdh1995/vimium-c#1192] [Ideas]
Hi,

Is it possible for Vimium-C to do something like tridactyl `seteditorcmd`?
It will open the current text editor, and save to the input box when done.

- https://github.com/tridactyl/tridactyl/blob/eb1141b8c89472225a45c8c2d39cc8bc2ab26ee5/.tridactylrc#L111
- https://gist.github.com/BurntSushi/393546a65db38d57cedcfd72c6d89bf3#file-tridactylrc-L13

Thanks a lot for Vimium-C 🥞

**Comment by gdh1995** [comment]
No the feature requires a very uncommon feature "nativeMessaging", and if a user wants to use it, a system-level software must be installed manually - see https://github.com/tridactyl/tridactyl?tab=readme-ov-file#extra-features-through-native-messaging .

Vimium C is not designed to work with any out-of-browser softwares, to keep it easy to install and use, protect user privacy and then keep itself trustworthy.

**Reply by azzamsa** [reply]
Thank you for Vimium C! 🥞
